kolibri/plugins/video_mp4_render/assets/src/vue/videojs-replay-forward-btns.js

```diff
@@ -21,21 +21,7 @@ class ForwardButton extends videojsButton {
   }
 }
 
-class TogglePlayButton extends videojsButton {
-  buildCSSClass() {
-    return 'vjs-control vjs-button videotoggle';
-  }
-  handleClick() {
-    const player = this.player();
-    this.toggleClass('videopaused');
-    if (player.paused()) {
-      player.play();
-    } else {
-      player.pause();
-    }
-  }
-}
-
+ReplayButton.prototype.controlText_ = 'Replay';
```

Comment:
How VideoJS does it: https://github.com/videojs/video.js/blob/master/docs/guides/languages.md